### PR TITLE
PDF reorder pull request.

### LINKED ISSUE #610: Pdf reordering using python script.
**Is your feature request related to a problem? Please describe.**
At times we get a pdf that is not ordered properly but we know the pages which have are out of order  and it is tough to reorder it.

**Describe the solution you'd like**
A python script that will take an input of pages which are unordered and where these unordered pages should go and parse it into dictionary.
Using this dictionary and a python library I will reorder the PDF.

**Describe alternatives you've considered**

I couldn't find any good free application on playstore for android or on the internet for windows.
Using this script I would be able to do the reordering with minimum hassle 

**Additional context**

### PR DESCRIPTION
**Pull request by Rahul555-droid**
# Description

The pdf reorder script. Did the changes to the readme file and moved images to imgur.

Fixes #610 

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Rahul555-droid** [comment]
@HarshCasper Sorry dude was busy with exams and also my laptop had some issues.
Created a new local repo and i am sending the PR again.

This issue and PR has stretched way long lol

I think this should work :)

File: Python/PDF_reorder/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Setup instructions
+
+- We use **pdfrw** library for this script. 
+- After downloading the folder switch to the folder's location on your commmand line.
+- To download required libraries enter `pip -r requirements.txt` in command line
+- To run the script `python3 pdf_reorder_.py` and then enter the required information.
+- After running the script it will create a modified file in the same folder as that of the original pdf file.
+
+## About the script pdf_reorder_.py
```
Comment (HarshCasper) [inline Python/PDF_reorder/README.md:13]: Just keep `About the Script`

File: Python/PDF_reorder/pdf_reorder_.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    dic = {curr: new for curr, new in lst}
+    # now I have sorted the dic to the required needs of reorder function
+    return dic
+
+
+"""
+The function reorder takes two arguments path and dic
+path is the path of the source pdf file which is in wrong
+order
+"""
```
Comment (HarshCasper) [inline Python/PDF_reorder/pdf_reorder_.py:29]: That's not how Docstrings are used. Refer this: https://www.programiz.com/python-programming/docstrings

File: Python/PDF_reorder/pdf_reorder_.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def reorder(path, dic):
+    # create a pdf object using PdfReader that could be read
+    pdf_obj = PdfReader(path)
+    # pdf_obj.pages attribute gives the length of the pages in pdf
+    total_pages = len(pdf_obj.pages)
+    print("Total Pages in PDF are:", total_pages)
+    # Initialising the writer object using the PdfWriter class,from this we would create a new modified Pdf
```
Comment (HarshCasper) [inline Python/PDF_reorder/pdf_reorder_.py:38]: Reduce the length of the line

File: Python/PDF_reorder/pdf_reorder_.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    """
+    store the input in a list and then convert the input string into
+    using map function to convert the data in lists into int values
+    """
+    lst = list(map(lambda x: [int(x[0]), int(x[1])],
+                   [input().split(',') for _ in range(n)]))
+    # Swapping the position of the lst values to better parse it in dictionary
+    lst = [[x[1], x[0]] for x in lst]
+    lst.sort(key=lambda x: x[0])
+    dic = {curr: new for curr, new in lst}
```
Comment (HarshCasper) [inline Python/PDF_reorder/pdf_reorder_.py:20]: Unnecessary use of comprehension

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
Are you working on this @Rahul555-droid ?

**Comment by HarshCasper** [comment]
Closing this PR because of no updates.

**Pull request closed without merging**